src/apps/Altinn.Authorization/test/Altinn.Authorization.Tests/ResourceRegistry_DecisionTests.cs

```diff
@@ -258,6 +258,59 @@ public async Task PDP_Decision_ResourceRegistry0004()
             AssertionUtil.AssertAuthorizationEvent(eventQueue, expectedAuthorizationEvent, Times.Once());
         }
 
+        [Fact]
+        public async Task PDP_Decision_ResourceRegistry_RequestConsent_ValidAccessList()
```

Comment:
mangler test dekning for autorisasjon basert på fnr, userid, partyid, orgnr.

Siden integrasjonen mot accesslist gjøres på PartyUuid skal det også støtte Systembruker, SI-bruker?

Reply:
Slik det er nå er det kun orgnr som kan legges på accessList. Så frem til det endres vil det ikke være mulig